QMLComponents/analysisbase.h

```diff
@@ -27,6 +28,7 @@ class AnalysisBase : public QObject
 	virtual const std::string & module()									const	{ return emptyString;		}
 	virtual const std::string & name()										const	{ return emptyString;		}
 	virtual const std::string & title()										const	{ return emptyString;		}
+	virtual const Version & version()										const	{ return AppInfo::version;	}
```

Comment:
Doesnt this mean that the versionnumbers in R-only mode will be totally wrong?

Reply:
As in, with the current code all the stuff that has only access to AnalysisBase will always return whatever was the version of JASP when jaspBase gets compiled right?

Why not just add `_version` in AnalysisBase and not have an override in Analysis and then just make sure a value gets set into it.